docs/notebooks/Migration_Guide_beginner.ipynb

```diff
@@ -0,0 +1,448 @@
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "H50AxL16EWI_"
+      },
+      "source": [
+        "Then we need to modify the `label_func`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "y4WBER0AEa43"
+      },
+      "source": [
+        "function label_func(dataframe)\n",
+        "  return(\n",
+        "    mapobs(Datasets.loadfile(df[!,\"path\"]), df[!,\"class\"]),\n",
+        "  )\n",
+        "  end\n",
+        "\n",
+        "image_data_container = label_func(df)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tYDG9X4wE_Rz"
+      },
+      "source": [
+        "This version of `label_func` will return the `image_data_container` itself. So no need to call `mapobs`"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "YF9FubrmFiDw"
+      },
+      "source": [
+        "# Learner"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bdx1u49NFxjj"
+      },
+      "source": [
+        "Learner is the object that binds the data loaders with the model and sets the other hyper parameters like loss function, optimizers, etc. There is `Learner()` function which is similar to fastai's `cnn_learner` function. The below python code can be translated as,\n",
+        "\n",
+        "```\n",
+        "learn = cnn_learner(dls, resnet34, metrics=error_rate)\n",
+        "\n",
+        "#to fine tune\n",
+        "learn.fine_tune(1)\n",
+        "\n",
+        "#to fit one cycle\n",
+        "learn.fit_one_cycle(1)\n",
+        "```"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "xzh7bif3G-HG"
+      },
+      "source": [
+        "model = Models.resnet34(pretrained = true)\n",
+        "metric = Metrics(Metric(Flux.error_rate))\n",
+        "learn = Learner(model, dls, lossfn , metric)\n",
+        "\n",
+        "# to finetune\n",
+        "finetune!(learn, 1)\n",
+        "\n",
+        "# to fit one cycle\n",
+        "fitonecycle!(learn,1)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fHifey18IVqf"
+      },
+      "source": [
+        "However, `Learn` method makes us to define every parameters manually. Like we should explicitly convert our data container object to `dataloader` object and pass them. \n",
+        "\n",
+        "Instead we can cut the complexity by using `methodlearner` function where everything can be passed as hyperparameters. The cool thing about `methodlearner` is it takes the data container object and implicitly converts it into dataloader, where the batch size, validation split can be customized if we need. Therefore the above code can be converted into, "
+      ]
+    },
```

Comment:
I would just delete this section. The comparison to `Learner` isn't exact, and I think `cnn_learner` is more closely compared to `methodlearner`.